asserter/asserter.go

```diff
@@ -78,6 +79,7 @@ func NewServer(
 	callMethods []string,
 	mempoolCoins bool,
 	validationFilePath string,
+	relatedOpsEnabled bool,
```

Comment:
We should put this as a part of validation config rather than create a new parameter. This breaks the existing usage.

Reply:
so we want to add this to the validations struct along with `enabled`, `chainType` etc.?

Reply:
Yep. the reason we added that validation file is to use and extend with more functionalities.

Reply:
got it! we'd only be able to check/throw an error for account based chains then

Reply:
opened a new PR: https://github.com/coinbase/rosetta-sdk-go/pull/355

Reply:
I mean we could so this check on both. Right now we only do for account based but we can change the code to have for both UTXO and account.